sci-astronomy/galaxy/galaxy-2.2-r1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit fdo-mime flag-o-matic toolchain-funcs
+
+# probably change every release
+PID="1/3/0/3/13035936"
+
+DESCRIPTION="Stellar simulation program"
+HOMEPAGE="http://www.kornelix.com/galaxy.html"
+SRC_URI="http://www.kornelix.com/uploads/${PID}/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE=""
+
+DEPEND="x11-libs/gtk+:3"
+RDEPEND="${DEPEND}
+	x11-misc/xdg-utils"
+
+pkg_setup() {
+	tc-export CXX
+	append-cxxflags -pthread
```

Comment:
This is also a matter of choice, but in general in Gentoo we prefer to set flags in `src_configure`, before calling the configure script. So rename this phase function to `src_configure`, add `default` at the bottom, and move the whole thing below `src_prepare`.